src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ManageContent/ReleaseNoteService.cs

```diff
@@ -48,7 +49,9 @@ public class ReleaseNoteService : IReleaseNoteService
                     {
                         On = saveRequest.On ?? DateTime.Now,
                         Reason = saveRequest.Reason,
-                        ReleaseId = release.Id
+                        ReleaseId = release.Id,
+                        Created = DateTime.UtcNow,
+                        CreatedById = _userService.GetUserId(),
```

Comment:
I see there are no tests for ReleaseNoteService at all, but we could at least test that CreatedById is retained as the original creating user rather than the user creating an amendment in  `ReleaseServiceAmendmentTests.AssertAmendedUpdateCorrect` when a release amendment is created.